Animal_Adoption/preproc/data_all.py

```diff
@@ -4,6 +4,10 @@
 import numpy as np
 from preproc.preproc_colors import perform_all_color_cleaning
 from preproc.preproc_intake_conditions import fix_age, drop_under_8_aged
+from preproc.target_relabled import breed_2class
```

Comment:
I'm pretty sure these imports won't work in all circumstances. Has this been tested by importing into a notebook?
`.target_relabeled` would probably work better since the files are in the same directory